### PR TITLE
Describe motivation in README

### PR DESCRIPTION
**Pull request by bmish**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,13 @@
 
 This plugin checks to ensure that dependencies are on consistent versions across a monorepo / yarn workspace. For example, every package in a workspace that has a dependency on `eslint` should specify the same version for it.
 
+Motivation:
```
Comment (ddzz) [inline README.md:8]:
```suggestion
## Motivation:
```

File: README.md [hunk 2]
```diff
@@ -5,6 +5,13 @@
...
+
+* **Supports a uniform developer experience.** Like shared code formatting standards, consistent dependency versions make it easier for developers to jump around a monorepo.
```
Comment (ddzz) [inline README.md:10]:
```suggestion
* **Supports a uniform developer experience.** Like shared code formatting standards, consistent dependency versions make it easier for developers to work among different workspaces in a monorepo.
```

File: README.md [hunk 3]
```diff
@@ -5,6 +5,13 @@
 
 This plugin checks to ensure that dependencies are on consistent versions across a monorepo / yarn workspace. For example, every package in a workspace that has a dependency on `eslint` should specify the same version for it.
 
+## Motivation:
```
Comment (ddzz) [inline README.md:8]:
```suggestion
## Motivation
```

This might still fail CI linting due to the colon.

File: README.md [hunk 4]
```diff
@@ -5,6 +5,14 @@
 
 This plugin checks to ensure that dependencies are on consistent versions across a monorepo / yarn workspace. For example, every package in a workspace that has a dependency on `eslint` should specify the same version for it.
 
+## Motivation
+
+* **Supports a uniform developer experience.** Like shared code formatting standards, consistent dependency versions make it easier for developers to jump around and work across packages in a monorepo.
+* **Discourages piecemeal upgrades.** When possible, a dependency should be upgraded everywhere it exists in a monorepo at the same time, and upgrades are simpler when fewer versions are present.
+* **Reduces unexpected behavior.** Having multiple, older versions present can lead to confusion, i.e. a bug present in some parts of a monorepo but not others.
```
Comment (ddzz) [inline README.md:12]:
```suggestion
* **Reduces unexpected behavior.** Having multiple, older versions present can lead to confusion, e.g., a bug may be present in some parts of a monorepo but not others.
```

Comment (bmish) [reply README.md]: This package deals only with a single workspace at a time. A workspace may have many packages inside it.

File: README.md [hunk 5]
```diff
@@ -5,6 +5,14 @@
 
 This plugin checks to ensure that dependencies are on consistent versions across a monorepo / yarn workspace. For example, every package in a workspace that has a dependency on `eslint` should specify the same version for it.
 
+## Motivation
+
+* **Supports a uniform developer experience.** Like shared code formatting standards, consistent dependency versions make it easier for developers to jump around and work across packages in a monorepo.
+* **Discourages piecemeal upgrades.** When possible, a dependency should be upgraded everywhere it exists in a monorepo at the same time, and upgrades are simpler when fewer versions are present.
+* **Reduces unexpected behavior.** Having multiple, older versions present can lead to confusion, i.e. a bug present in some parts of a monorepo but not others, or uncertainty about which version wins out when served to the browser.
```
Comment (ddzz) [inline README.md:12]:
```suggestion
* **Reduces unexpected behavior.** Having multiple, older versions present can lead to confusion, e.g., a bug present in some parts of a monorepo but not others, or uncertainty about which version wins out when served to the browser.
```

**Review by ddzz — APPROVED** [review]

**Review by ddzz — APPROVED** [review]

**Pull request merged**